src/views/Overview/OverviewQuickLinks.vue

```diff
@@ -47,16 +42,21 @@ export default {
   components: {
     ChevronRight16
   },
+  created() {
+    this.getBmcTime();
+  },
+  computed: {
+    bmcTime() {
+      return this.$store.getters['global/bmcTime'];
+    }
+  },
+  methods: {
+    getBmcTime() {
+      this.$store.dispatch('global/getBmcTime');
+    }
+  },
   data() {
     return {
-      bmc: {
-        Elapsed: 1574782085071
-      },
-      logging: {
-        entry: {
-          Severity: 'Error'
-        }
-      },
       checked: false
```

Comment:
When we add functionality for this, I think the name should be more descriptive, something like `serverLEDChecked`